src/test/java/seedu/address/testutil/TodoBuilder.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.testutil;
+
+import seedu.address.model.task.repeatable.Event;
+import seedu.address.model.task.todo.Todo;
+
+/**
+ * A utility class to help with building Todo objects.
+ */
+public class TodoBuilder {
+
+    public static final String DEFAULT_DESCRIPTION = "Todo Description";
+    public static final Boolean DEFAULT_IS_DONE = false;
+
+    private String description;
+    private Boolean isDone;
+
+    /**
+     * Creates a {@code TodoBuilder} with the default details.
+     */
+    public TodoBuilder() {
+        description = DEFAULT_DESCRIPTION;
+        isDone = DEFAULT_IS_DONE;
+    }
+
+    /**
+     * Initializes the TodoBuilder with the data of {@code todoToCopy}.
+     */
+    public TodoBuilder(Event eventToCopy) {
+        description = eventToCopy.getDescription();
+        isDone = eventToCopy.getIsDone();
+    }
+
+    /**
+     * Sets the {@code Description} of the {@code Todo} that we are building.
+     */
+    public TodoBuilder withDescription(String description) {
+        this.description = description;
+        return this;
+    }
+
+
+    /**
+     * Sets the {@code isDone} status of the {@code Todo} that we are building.
+     */
+    public TodoBuilder withIsDone(Boolean isDone) {
+        this.isDone = isDone;
+        return this;
+    }
+
+    /**
+     * Builds the {@code Todo} object.
+     * @return {@code Todo}.
+     */
+    public Todo build() {
+        return new Todo(description, isDone);
+    }
+
+}
```

Comment:
Ahhh, OK I think that is all I can spot for now. For cases to test, I cannot think of other cases to add.